content/docs/getting-started/hello-world.md

```diff
@@ -37,48 +35,42 @@ and the crates and types into scope in `main.rs`:
 extern crate tokio;
 
 use tokio::io;
-use tokio::net::TcpListener;
+use tokio::fs::File;
 use tokio::prelude::*;
 # fn main() {}
 ```
 
-# Writing the server
+# Creating the file
 
-The first step is to bind a `TcpListener` to a local port. We use the
-`TcpListener` implementation provided by Tokio.
+The first step is to create the `File`. We use the `File` implementation provided by Tokio.
```

Comment:
Could it be worth mentioning what the difference is between `std::fs::File` and `tokio::fs::File`?

I'd follow what the [documentation](https://docs.rs/tokio/0.1.11/tokio/fs/file/struct.File.html) says.

> The first step is to create the `File`. We use the `File` implementation provided by Tokio. This is a specialized version of `std::fs::File` for usage from the Tokio runtime.

It's probably worth pointing out that once we have a`tokio::fs::File` we will have to interact with it with tokio-specific functions. E.g. [`tokio::io::Write::write_all`](https://tokio-rs.github.io/tokio/tokio/io/trait.Write.html#method.write_all) vs [`std::io::Write::write_all`](https://doc.rust-lang.org/std/io/trait.Write.html#method.write_all).